fix: Set SentryEvent timestamp to be the time the event was created (#1629)
getsentry/sentry-cocoa: CHANGELOG.md

# Changelog

## Unreleased

> - fix: Set SentryEvent timestamp to be the time the event was created (#1629)

Reviewer: `m`: Can we please limit the git commit message, the changelog entry, and the title of the PR to 50 characters, please? 
```suggestion
- fix: Set timestamp in init of event (#1629)
```
